index.js

```diff
@@ -11,8 +11,39 @@ const COLORS = 'https://nt-cdn.s3.amazonaws.com/colors.json';
  * @returns Promise
  */
 const fetchColors = ({ name, hex, compName, compHex }) => {
-  throw Error('Not implemented');
+
+  const filter = fetch(COLORS)
+    .then(response => response.json())
+    .then(colors => {
+      if (name != undefined) {
+        return colors.filter(
+          color => color.name.toLowerCase() === name.toLowerCase()
```

Comment:
Remember to DRY your code. The string comparison could be moved to its own function so if the requirement changed to be case sensitive you only need to do it in one place.

Reply:
Thanks! I will do it on the future 😄